modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -181,11 +182,12 @@ object ElasticQuery {
     override def toJson: Option[(String, Json)] = None
   }
 
-  private[elasticsearch] final case class RangeQuery[A, LB <: LowerBound, UB <: UpperBound] private (
+  private[elasticsearch] final case class RangeQuery[A, LB <: LowerBound, UB <: UpperBound](
```

Comment:
Why did you omit `private`?

Reply:
I removed private because of boost default argument would raise a warning: `private default argument in object RangeQuery is never used during` during compilation on scala 2.12.17. 
I saw that you removed all default arguments on case class, I'll make the changes !